### PR TITLE
Added Hot Reload directions for Django and Flask

### PR DESCRIPTION
**Pull request by ucheNkadiCode**
We're adding to our documentation to inform users of how to enable hot reloading in their containerized application. 

**Please** wait for a sign off from @haniamr

### CODE REVIEW

File: docs/containers/debug-python.md [hunk 1]
```diff
@@ -107,7 +107,123 @@ Here is an example of using `dockerServerReadyAction` to launch the browser to o
 
 - `uriFormat`: By default, the Docker extension will open the main page of the browser (however that is determined by the application). If you want the browser to open a specific page like the example above, the `uriFormat` property of the [dockerServerReadyAction](debug-common.md#dockerServerReadyAction-object-properties) object should be set to a format string with two string tokens to indicate the protocol and port substitution.
 
-### How to debug your app with Gunicorn
+## How to enable hot reloading in Django or Flask apps
+
+When you select **Docker: Add Docker Files to Workspace** for Django or Flask, we provide you a Dockerfile and `tasks.json` configured for static deployment. Each time you make changes to your app code, you will need to rebuild and re-run your container. Hot reloading allows you to visualize changes in your app code as your container continues to run. Enable hot reloading with these steps:
```
Comment (ghogen) [inline docs/containers/debug-python.md:112]:
```suggestion
When you select **Docker: Add Docker Files to Workspace** for Django or Flask, we provide you a Dockerfile and `tasks.json` configured for static deployment. Each time you make changes to your app code, you need to rebuild and re-run your container. Hot reloading allows you to visualize changes in your app code as your container continues to run. Enable hot reloading with these steps:
```

File: docs/containers/debug-python.md [hunk 2]
```diff
@@ -107,7 +107,123 @@ Here is an example of using `dockerServerReadyAction` to launch the browser to o
...
+          "runserver",
+          "0.0.0.0:8000",
+        ],
+      }
+      ...
+    }
+    ```
+
+1. Select the **Docker: Python – Django** launch configuration and hit `kb(workbench.action.debug.start)` to build and run your container.
+1. Modify and save an HTML file.
```
Comment (haniamr) [inline docs/containers/debug-python.md:165]: This doesn't have to be an HTML file, could be a python file as well. So I would maybe reword it to `Modify and save any file`.

File: docs/containers/debug-python.md [hunk 3]
```diff
@@ -107,7 +107,123 @@ Here is an example of using `dockerServerReadyAction` to launch the browser to o
...
+
+### For Flask Apps
+
+1. In the Dockerfile, comment out the line that adds app code to the container.
+
+    ``` dockerfile
+    #ADD . /app
+    ```
+
+1. Within the `docker-run` task in the `tasks.json` file, create a new dockerRun attribute with a volumes property. This will create a mapping from the current workspace folder (app code) to the `/app` folder in the container.
```
Comment (haniamr) [inline docs/containers/debug-python.md:176]: If users are using the default scaffold, then there will be an existing `dockerRun` attribute.

File: docs/containers/debug-python.md [hunk 4]
```diff
@@ -107,7 +107,123 @@ Here is an example of using `dockerServerReadyAction` to launch the browser to o
...
+    {
+      ...
+      "dockerRun": {
+        "volumes": [
+          {
+            "containerPath": "/app", "localPath": "${workspaceFolder}"
+          }
+        ]
+      },
+      "python": {
```
Comment (haniamr) [inline docs/containers/debug-python.md:212]: This looks like a Django configuration not a Flask one.

File: docs/containers/debug-python.md [hunk 5]
```diff
@@ -107,7 +107,123 @@ Here is an example of using `dockerServerReadyAction` to launch the browser to o
...
+          "runserver",
+          "0.0.0.0:8000",
+        ],
+      }
+      ...
+    }
+    ```
+
+1. Select the **Docker: Python – Flask** launch configuration and hit `kb(workbench.action.debug.start)` to build and run your container.
+1. Modify and save an HTML file.
```
Comment (haniamr) [inline docs/containers/debug-python.md:223]: Same idea as above, could be any file type.

File: docs/containers/debug-python.md (hunk 2 above)
Comment (ucheNkadiCode) [reply docs/containers/debug-python.md]: Made this fix

File: docs/containers/debug-python.md (hunk 3 above)
Comment (ucheNkadiCode) [reply docs/containers/debug-python.md]: Changed the wording to "edit the existing dockerRun attribute..."

File: docs/containers/debug-python.md (hunk 4 above)
Comment (ucheNkadiCode) [reply docs/containers/debug-python.md:216]: Thanks for that catch! I made the necessary changes

File: docs/containers/debug-python.md (hunk 5 above)
Comment (ucheNkadiCode) [reply docs/containers/debug-python.md]: Thanks again, made that change

File: docs/containers/debug-python.md (hunk 4 above)
Comment (haniamr) [reply docs/containers/debug-python.md:216]: Just one more minor thing, I'd change the args to be in the new format:
```
"args": [
          "run",
          "--host", "0.0.0.0",
          "--port", "5000"
        ]
```
